lib/std/Thread/Condition.zig

```diff
@@ -1,411 +1,538 @@
-//! A condition provides a way for a kernel thread to block until it is signaled
-//! to wake up. Spurious wakeups are possible.
-//! This API supports static initialization and does not require deinitialization.
-
-impl: Impl = .{},
+//! Condition variables are used in tangent with a Mutex to efficiently wait for an arbitrary condition to occur.
```

Comment:
Nitpick: does `in tangent with` mean the same thing as `together with`? if yes, I would use the latter phrase instead of the former.